src/test/coherentsolutions.web.test/java/YandexTest.java

```diff
@@ -0,0 +1,48 @@
+import Locators.Locators;
+import Managers.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static Utils.PauseUtils.pause;
+
+public class YandexTest {
+
+    private WebDriver driver;
+
+    private final String PASSWORD = "testik1234";
+    private final String LOGIN = "katetestng";
+
+    @BeforeMethod
+    public void addSettings() {
+        driver = WebDriverManager.initDriver();
+    }
+
+    @Test
```

Comment:
It would be nice if you add description parameter to the annotation which describe what you do in this test